src/fiatconnect/types.ts

```diff
@@ -11,3 +11,15 @@ export interface FiatAccountSchemaCountryOverrides {
     }
   }
 }
+
+export class FiatConnectTxError extends Error {
+  txPossiblyPending: boolean
+  error: any
+
+  constructor(message: string, txPossiblyPending: boolean, error: any) {
+    super(message)
+    Object.setPrototypeOf(this, FiatConnectTxError.prototype)
+    this.txPossiblyPending = txPossiblyPending
```

Comment:
seems like you could just compute this value here, instead of taking it as a parameter. but maybe it was useful for testing to do it this way, i haven't looked at the tests closely

Reply:
It's simpler for testing to do it outside of the error constructor. After moving the timeout check into the helper function though, we can just do this inline when passing args to the constructor.